path/posix/basename.ts

```diff
@@ -24,37 +23,15 @@ import { fromFileUrl } from "./from_file_url.ts";
  * assertEquals(basename("/home/user/Documents/image.png", ".png"), "image");
  * ```
  *
- * @param path The path to extract the name from.
- * @param suffix The suffix to remove from extracted name.
- * @returns The extracted name.
- */
-export function basename(path: string, suffix?: string): string;
-/**
- * Return the last portion of a `path`.
- * Trailing directory separators are ignored, and optional suffix is removed.
- *
- * @experimental **UNSTABLE**: New API, yet to be vetted.
- *
- * @example Usage
- * ```ts
- * import { basename } from "@std/path/posix/basename";
- * import { assertEquals } from "@std/assert";
- *
- * assertEquals(basename("/home/user/Documents/"), "Documents");
- * assertEquals(basename("/home/user/Documents/image.png"), "image.png");
- * assertEquals(basename("/home/user/Documents/image.png", ".png"), "image");
- * assertEquals(basename(new URL("file:///home/user/Documents/image.png")), "image.png");
- * assertEquals(basename(new URL("file:///home/user/Documents/image.png"), ".png"), "image");
- * ```
+ * Note: If you are working with file URLs,
+ * use the new version of `basename` from `@std/path/posix/unstable-basename`.
  *
  * @param path The path to extract the name from.
  * @param suffix The suffix to remove from extracted name.
  * @returns The extracted name.
  */
-export function basename(path: string | URL, suffix?: string): string;
-export function basename(path: string | URL, suffix = ""): string {
-  path = path instanceof URL ? fromFileUrl(path) : path;
-  assertArgs(path, suffix);
+export function basename(path: string, suffix?: string): string {
+  assertArgs(path, suffix ?? "");
```

Comment:
Ditto for the other functions
```suggestion
export function basename(path: string, suffix = ""): string {
  assertArgs(path, suffix);
```